tests/checks/mock/test_splunk_topology.py

```diff
@@ -386,10 +392,78 @@ def test_checks(self):
         try:
             self.run_check(config, mocks={
                 '_dispatch_saved_search': _mocked_dispatch_saved_search,
-                '_search': _mocked_search
+                '_search': _mocked_search,
+                '_saved_searches': _mocked_saved_searches
             })
         except CheckException:
             thrown = True
         self.assertTrue(thrown, "Retrieving FATAL message from Splunk should throw.")
 
         self.assertEquals(self.service_checks[0]['status'], 2, "service check should have status AgentCheck.CRITICAL")
+
+class TestSplunkWildcardTopology(AgentCheckTest):
+    """
+    Splunk check should work with component and relation data
+    """
+    CHECK_NAME = 'splunk_topology'
+
+    def test_checks(self):
+        self.maxDiff = None
```

Comment:
If I recall correctly it governs it governs the maximum size of the diff when an assertion fails